app/src/bcsc-theme/features/auth/LockoutScreen.tsx

```diff
@@ -0,0 +1,131 @@
+  ThemedText,
+  TOKENS,
+  useServices,
+  useTheme,
+} from '@bifold/core'
+import { CommonActions } from '@react-navigation/native'
+import { StackNavigationProp } from '@react-navigation/stack'
+import { useCallback, useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, View } from 'react-native'
+import { isAccountLocked, removeAccount } from 'react-native-bcsc-core'
+
+const formatTime = (seconds: number): string => {
+  const minutes = Math.floor(seconds / 60)
+  const remainingSecs = seconds % 60
+
+  if (minutes > 0) {
+    return `${minutes} minute${minutes === 1 ? '' : 's'} ${remainingSecs} second${remainingSecs === 1 ? '' : 's'}`
+  }
+  return `${remainingSecs} second${remainingSecs === 1 ? '' : 's'}`
+}
+
+interface LockoutScreenProps {
+  navigation: StackNavigationProp<BCSCAuthStackParams, BCSCScreens.Lockout>
+}
+
+export const LockoutScreen = ({ navigation }: LockoutScreenProps) => {
+  const { TextTheme, Spacing } = useTheme()
+  const { t } = useTranslation()
+  const [logger] = useServices([TOKENS.UTIL_LOGGER])
+  const [remainingSeconds, setRemainingSeconds] = useState<number>(0)
+  const [shouldNavigateBack, setShouldNavigateBack] = useState(false)
+
+  const styles = StyleSheet.create({
+    hr: {
+      borderBottomWidth: 1,
+      borderBottomColor: TextTheme.normal.color,
+      width: '100%',
+    },
+  })
+
+  useEffect(() => {
+    const checkLockStatus = async () => {
+      try {
+        const { locked, remainingTime } = await isAccountLocked()
+
+        if (!locked || remainingTime <= 0) {
+          setShouldNavigateBack(true)
+          return
+        }
+
+        // remainingTime is already in seconds from native code
+        setRemainingSeconds(remainingTime)
+      } catch (error) {
+        const strErr = error instanceof Error ? error.message : String(error)
+        logger.error(`Error checking lock status: ${strErr}`)
+      }
+    }
+
+    checkLockStatus()
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
+
+  useEffect(() => {
+    if (shouldNavigateBack) {
+      navigation.dispatch(
+        CommonActions.reset({
+          index: 0,
+          routes: [{ name: BCSCScreens.EnterPIN }],
+        })
+      )
+    }
+  }, [shouldNavigateBack, navigation])
+
+  useEffect(() => {
+    if (remainingSeconds <= 0) {
+      return
+    }
+
+    const intervalId = setInterval(() => {
+      setRemainingSeconds((prev) => {
+        const newValue = prev - 1
+        if (newValue <= 0) {
+          setShouldNavigateBack(true)
+          return 0
+        }
+        return newValue
+      })
+    }, 1000)
+
+    return () => clearInterval(intervalId)
+  }, [remainingSeconds])
+
+  const onPressRemoveAccount = useCallback(async () => {
+    try {
+      await removeAccount()
+    } catch (error) {
+      const strErr = error instanceof Error ? error.message : String(error)
+      logger.error(`Error removing account: ${strErr}`)
```

Comment:
I don't love using Hungarian notation in TypeScript. We have a type system which will tell us it's a string. Would be better to use errorMessage or something if it's not actually an "Error" object.